src/main/java/org/repodriller/persistence/csv/CSVFile.java

```diff
@@ -45,15 +50,42 @@ public CSVFile(String path, String name) {
 
 	public CSVFile(String path, String name, boolean append) {
 		this(verifyPath(path) + name, append);
+	}		
+	
+	public CSVFile (String path, String name , String [] header) {
+		this(verifyPath(path) + name, header);
+	}
+	
+	public CSVFile (String fileName, String [] header) {
+		this(fileName,header,false);
 	}
 
+	public CSVFile(String fileName, String[] header, boolean append ){
+		this.header = header;
+		try {
+			if (header == null)
+				ps = new PrintStream(new FileOutputStream(fileName, append));
```

Comment:
this line can be outside the `if`, and then you don't need the repetition in the `else`